auto_tests/tox_many_tcp_test.c

```diff
@@ -54,19 +51,25 @@ static void test_many_clients_tcp(void)
         struct Tox_Options *opts = tox_options_new(nullptr);
 
         if (i == 0) {
-            tox_options_set_tcp_port(opts, TCP_RELAY_PORT);
+            tox_options_set_tcp_port(opts, tcp_relay_port);
         } else {
             tox_options_set_udp_enabled(opts, false);
         }
 
         index[i] = i + 1;
-        toxes[i] = tox_new_log(opts, nullptr, &index[i]);
+        Tox_Err_New err;
+        toxes[i] = tox_new_log(opts, &err, &index[i]);
+        if (i == 0 && err == TOX_ERR_NEW_PORT_ALLOC) {
+            --i;
+            ++tcp_relay_port;
+            continue;
```

Comment:
This also needs `tox_options_free(opts)`.

Reply:
Good catch. Done.